spec.emu

```diff
@@ -77,36 +77,36 @@ contributors: Google, Ecma International
         </tr>
         <tr>
           <td>*"ethiopic"*</td>
-          <td>*"ethiopic"*</td>
-          <td>*"incar"*</td>
+          <td>*"am"*</td>
+          <td>*"ethiopic-amete-mihret"*, *"incar"*</td>
           <td>*1*<sub>𝔽</sub></td>
           <td>*+&infin;*<sub>𝔽</sub></td>
         </tr>
         <tr>
           <td>*"ethiopic"*</td>
-          <td>*"ethioaa"*</td>
-          <td>*"ethiopic-amete-alem"*, *"mundi"*</td>
+          <td>*"aa"*</td>
+          <td>*"ethioaa"*, *"ethiopic-amete-alem"*, *"mundi"*</td>
```

Comment:
```suggestion
          <td>*"mundi"*</td>
```
(`ethiopic-amete-alem` no longer in https://github.com/unicode-org/cldr/pull/4519)

Reply:
hmm, we could consider dropping incar and mundi, too. I want to wait for confirmation from Mark.

Reply:
I think we should go ahead without dropping `incar` and `mundi` and we can revisit that after you get such confirmation.